api-reference/authentication/get-users31.markdown

```diff
@@ -78,6 +78,7 @@ successful call, responds with
       "PrimaryEmail": "johndoe@gmail.com",
       "EmployeeID": "johndoe@gmail.com",
       "ID": "99BFFFC3-C0BE-44FF-A441-AE1FFFFFF75B8",
+      "Emails": {"PrimaryEmail","VerifiedEmail,"email2","email3","email4","email5"},
```

Comment:
These JSON objects are invalid. Is this supposed to be an array?

Reply:
Hey, yes those are now returned as an array.  How should that be formatted?

Reply:
Should be:

```
"Emails": ["PrimaryEmail", "VerifiedEmail", "email2", "email3", "email4", "email5"],
```